dev-libs/ixion/ixion0.18-0.19.0.recipe

```diff
@@ -100,7 +100,7 @@ PATCH()
 BUILD()
 {
 	autoreconf -fi
-	export CXXFLAGS="-std=c++17"
+	export CXXFLAGS="-std=c++17 -O2"
```

Comment:
```suggestion
	export CXXFLAGS="-std=c++17 -O2 -g"
```

The default is "-O2 -g" and this contains a debuginfo package, so it should retain that.